pkg/scheduler/plugins/gang/gang.go

```diff
@@ -155,7 +155,7 @@ func (gp *gangPlugin) OnSessionOpen(ssn *framework.Session) {
 	jobStarvingFn := func(obj interface{}) bool {
 		ji := obj.(*api.JobInfo)
 		occupied := ji.WaitingTaskNum() + ji.ReadyTaskNum()
-		if ji.CheckTaskMinAvailablePipelined() && occupied < ji.MinAvailable {
+		if !ji.CheckTaskMinAvailablePipelined() && occupied < ji.MinAvailable {
```

Comment:
the fix is good to me. is there any e2e or ut against the case? 
The mistake code pr should trigger the e2e failure.